src/views/Home/index.scss

```diff
@@ -0,0 +1,7 @@
+.home {
```

Comment:
`.easi-home` perhaps. just `.home` seems to general asking for CSS name spacing issues.